test/e2e/run_e2e.sh

```diff
@@ -231,6 +241,37 @@ if [ "$stackset_e2e" = true ]; then
     E2E_NAMESPACE="${namespace}" ./stackset-e2e -test.parallel 20
 fi
 
+if [ "$loadtest_e2e" = true ]; then
+  >&2 echo "collect loadtest e2e data"
+  prometheus=$(kubectl -n loadtest-e2e get ing prometheus -o json | jq -r '.spec.rules[0].host')
```

Comment:
Maybe use hardcoded cluster.local histname?

Reply:
no because we can't query from the cluster were the e2e orchestrator is executed to internal domains of the e2e cluster